### ISSUE TITLE
Layout does not collapse on iPhone

### ISSUE THREAD

**Issue by pattyland** [OSAS/strapping-mediawiki#29]
When I'm using Chrome on my iPhone the main Layout does not collapse.

I see the header and collapsed menus, but the body remains a few hundres pixel wide, which results in scrolling and pinching!

**Comment by garrett** [comment]
That's really odd. I don't have an iPhone, but I'll check things in my various Android browsers and on my iPad 1 / iPod Touch 1 (both of which are seriously outdated). 

Hopefully I'll see and fix something that works on the iPhone too.

**Comment by garrett** [comment]
I think this is related to #24. I do know that responsive was working earlier. I'll make sure responsiveness works, and it should hopefully fix rendering on the iPhone too.

**Comment by garrett** [comment]
Okay, after lots of debugging _everything_, I've finally traced it down to a broken implementation of MediaWiki ResourceLoader and the way it handles media queries (or doesn't, properly, that is). I have a work-around fix landing in the repo shortly.

I've also tested it on my iPod Touch first generation, and can confirm it works in that really old version of Safari, so it should probably work in a more modern one too. In addition, I charged up my iPad 1 and checked it there too. It works there for me as well.
